lib/ExtUtils/MakeMaker/Locale.pm

```diff
@@ -47,7 +47,10 @@ sub _init {
             unless (defined &GetInputCP) {
                 eval {
                     require Win32;
-                    eval { Win32::GetConsoleCP() };
+                    eval {
+                        local $SIG{__WARN__} = sub {} if ( $] < 5.014 ); # suppress deprecation warning for inherited AUTOLOAD of Win32::GetConsoleCP()
```

Comment:
`local $^W if …`?

Reply:
Sure, I'm happy to rewrite it into that idiom. I see that that formulation is already is use elsewhere in the module (ie, *lib/ExtUtils/MakeMaker/version/vpp.pm*).

Reply:
Nope, the suppression is requiring the full $`SIG{__WARN__}...`.
I'm just going to leave it in full verbosity (and function).